test/integration/test-video-players-helper.js

```diff
@@ -268,7 +268,7 @@ export function runVideoPlayerIntegrationTests(
             player = r;
           });
         });
-        it('when the video ends', function () {
+        it.only('when the video ends', function () {
```

Comment:
Pretty sure we don't want to leave this change in.

Reply:
👍  Caught that right after creating the commit, already fixed